src/vs/workbench/contrib/chat/common/chatService.ts

```diff
@@ -104,6 +104,7 @@ export interface IChatResponseCommandFollowup {
 	commandId: string;
 	args?: any[];
 	title: string; // supports codicon strings
+	when?: string;
```

Comment:
Can you make this a `ContextKeyExpression` and call deserialize somewhere upstream on the when clause that came from the extension?

Reply:
I'll keep this as a string due to additional serialization support needed for the ContextKeyExpression types.